helm/chart-operator/templates/rbac.yaml

```diff
@@ -1,4 +1,144 @@
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
This won't work. With Helm 3 the security model changes.

Instead of running Tiller as cluster-admin we need to run chart-operator with this role.

https://github.com/giantswarm/chart-operator/blob/039d1d62f4ee3e5a35366be97c06343c590f44b4/helm/chart-operator/templates/rbac.yaml#L1-L14

Reply:
Thanks, I think this gets into while I resolve conflict files.